src/test/java/com/squareup/kotlinpoet/FunSpecTest.kt

```diff
@@ -347,4 +347,14 @@ class FunSpecTest {
       |}
       |""".trimMargin())
   }
+
+  @Test fun escapePunctuationInFunctionName() {
+    val funSpec = FunSpec.builder("with-hyphen")
+            .build()
```

Comment:
nit: please use 4 spaces for continuation indents

Reply:
sure. let me update it accordingly